src/createServer.js

```diff
@@ -1,11 +1,280 @@
 'use strict';
 
-// const express = require('express');
+const express = require('express');
 
 function createServer() {
   // Use express to create a server
   // Add a routes to the server
   // Return the server (express app)
```

Comment:
Delete comments all over the project. And do not leave such comments in your future test tasks.